website/content/docs/drivers/docker.mdx

```diff
@@ -1037,6 +1037,10 @@ host system.
   wait before cancelling an in-progress pull of the Docker image as specified in
   `infra_image`. Defaults to `"5m"`.
 
+- `image_pull_timeout` - (Optional) A default time duration that controls how long Nomad
+  will wait before cancelling an in-progress pull of the Docker image as specified
```

Comment:
```suggestion
  waits before cancelling an in-progress pull of the Docker image as specified
```

Style: [use present tense](https://github.com/hashicorp/tutorials/blob/main/STYLE_GUIDE.md#time)

Reply:
Thank you! For the review and the link!